Eevee.Sleep.Bot/Controllers/InGameAnnouncementController.cs

```diff
@@ -0,0 +1,34 @@
+using Eevee.Sleep.Bot.Controllers.Mongo.InGameAnnouncement;
+using Eevee.Sleep.Bot.Enums;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Eevee.Sleep.Bot.Controllers;
+
+[ApiController]
+[Route("/game/announcement")]
+public class InGameAnnouncementController(
+) : ControllerBase {
+    [HttpGet("{Locale}", Name = "GetInGameAnnouncements")]
+    public ActionResult<string> Get(string Locale) {
+        if (!Enum.TryParse<InGameAnnoucementLanguage>(Locale, true, out var language)) {
+            return BadRequest();
+        }
+
+        var announcement = InGameAnnouncememntIndexController.FindAllByLanguage(language);
+        return Ok(announcement.Select(x => x.ToApiResponse()));
+    }
+
+    [HttpGet("{Locale}/{AnnouncementId}", Name = "GetInGameAnnouncementDetails")]
```

Comment:
Locale may not be necessary because IDs are unique regardless of language

Reply:
Still keep the locale for easier searching and index creation. For example, EN website users won't care about ZH announcements.

Reply:
OK 👌 
Also added language_announcementId index